content/how-to-todo-in-ember-template-lint.md

```diff
@@ -0,0 +1,121 @@
+      "warn": 15,
+      "error": 30
+    }
+  }
+}
+```
+
+The default values will vary from team to team, but in general the idea is this: it should be reasonable enough for teams to methodically work on past issues, but not too long that they are forgotten about. _*It should feel like a little bit of a stretch, but not stressful.*_
+
+### Increasing specificity by rule
+
+It's also possible that you'd want to specify that a particular rule needs to be fix with more immediacy. In that case, you'd want to run that rule by itself. Let's use the `no-nested-interactive` rule as an example:
+
+```bash
+yarn ember-template-lint app/ --no-config-path --no-inline-config --rule 'no-nested-interactive:error' --update-todo --todo-days-to-warn=7 --todo-days-to-error=14
+```
+
+Let's break down what this command does:
+
+- `yarn ember-template-lint app/` : "run the linter on the app folder"
+- `--no-config-path` : "ignore the `.template-lintrc.js` file"
+- `--no-inline-config` : "also, ignore instances where the rule has been supressed inline"
+- `--rule 'no-nested-interactive:error'` : "specifically, only run the `no-nested-interactive` rule and throw an error if it finds these issues"
+- `--update-todo` : "create a todo if this issue is found"
+- `--todo-days-to-warn=7` : "in the created todo, automatically change the todo to a warning after 7 days"
+- `--todo-days-to-error=14` : "in the created todo, automatically change the warning to an error after 14 days"
+
+<div class="cta-note" style="max-width:100%">
+  <div class="cta-note-content">
+    <div class="cta-note-body p-2">
+      <div class="text-lg mb-1 hide-in-percy" data-test-es-note-heading="">Zoey says...accessibility!</div>
+      <div class="cta-note-message">
+        <p>If your company has decided to make more inclusive products, you may want to set the decay days based on the rule itself.</p>
+        <p>Typically, accessibility (a11y) bugs are divided up into categories based on impact to the user: blocker, critical, major and minor (results may vary).</p>
+        <p>Looking at the documentation for each the <a href="https://github.com/ember-template-lint/ember-template-lint/blob/master/lib/config/a11y.js">a11y-related rules</a> in <code>ember-template-lint</code>, you could then determine the rule's category, and determine the number of days to warn/error accordingly.</p>
+        <p>For more ideas about how this could work, ping the #topic-a11y channel in the <a href="https://discord.gg/emberjs">Ember Discord community chat</a>.</p>
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="" class="hide-in-percy cta-mascot m-2" style="max-height:100px" />
+  </div>
+</div>
+
+
+## Step 2: Plan For Accountability
+
+After running the linter and creating all of the todos, a `.lint-todo` folder will appear in the root directory of the project. This is where you'll find the specifics about what file is failing what rule, and you can fix those issues.
```

Comment:
```suggestion
After running the linter and creating all of the todos, a `.lint-todo` folder will appear in the root directory of the project. This is where you'll find the specifics about which file is failing which rule so that you can fix the issues.
```